src/glmfit.jl

```diff
@@ -115,7 +115,23 @@ function confint(obj::AbstractGLM, level::Real)
 end
 confint(obj::AbstractGLM) = confint(obj, 0.95)
 
-deviance(m::AbstractGLM)  = deviance(m.rr)
+deviance(m::AbstractGLM) = deviance(m.rr)
+
+function loglikelihood(m::AbstractGLM)
```

Comment:
The log likelihood may have terms that depend only on the samples or on the dispersion parameter. These drop out when computing the deviance since, as @andreasnoack says, it is defined as -2*the difference between the log likelihoods for the fitted model and the saturated model, but to compute the full log likelihood you would need to include them. I think the only way to do this is to compare the formulas for the log PDF and the deviance residual for the specific distribution and adjust the deviance as necessary. Also, when the distribution has a dispersion parameter, the log likelihood may depend upon how that dispersion parameter is estimated (method of moments vs. ML).